System.Security.Cryptography/AES.cs

```diff
@@ -0,0 +1,162 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    /// Represents the class for aes encryption/decryption
+    /// </summary>
+    public class AES
+    {
+        public enum EncryptionModes { ECB }
+        public EncryptionModes Mode { get; set; } = EncryptionModes.ECB;
+
+        /// <summary>
+        /// Encrypt the array of bytes.
+        /// </summary>
+        /// <param name="key">The secret key to use for the symmetric algorithm.</param>
+        /// <param name="data">array of bytes for encryption</param>
+        /// <returns></returns>
+        public byte[] Encrypt(byte[] key, byte[] data)
+        {
+            byte[] buf = null;
+
+            if (Mode == EncryptionModes.ECB)
+            {
+                buf = EncryptAesEcb(key, data);
+            }
+
+            return buf;
+        }
+
+        /// <summary>
+        /// Decrypt the array of bytes.
+        /// </summary>
+        /// <param name="key">The secret key to use for the symmetric algorithm.</param>
+        /// <param name="data">encrypted array of bytes for decryption</param>
+        /// <returns></returns>
+        public byte[] Decrypt(byte[] key, byte[] data)
+        {
+            byte[] buf = null;
+
+            if (Mode == EncryptionModes.ECB)
+            {
+                buf = DecryptAesEcb(key, data);
+            }
+
+            return buf;
+        }
+
+        /// <summary>
+        /// Encrypt the array of bytes In ECB Mode
+        /// </summary>
+        /// <param name="key">The secret key to use for the symmetric algorithm.</param>
+        /// <param name="data">Array of bytes for encryption</param>
+        /// <returns></returns>
```

Comment:
same remark here, even if it's a private function and even if intellisense comments are not mandatory, if you add them, please add them properly.